docs/_posts/2022-10-31-luau-semantic-subtyping.md

```diff
@@ -191,7 +191,7 @@ The other difference between Luau’s type system and off-the-shelf semantic sub
 The common case for wanting negated types is in typechecking conditionals:
 ```lua
 -- initially x has type T
-if (type(x) == “string”) then
+if (type(x) == "string") then
```

Comment:
Btw there's more of these fancy quotes, that's why I suggested running `sed`.

Reply:
I thought this was the only one in code. I want to keep the non-code ones.